vault/changelog.md

```diff
@@ -2,10 +2,36 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1659456790639
+updated: 1659968508042
 created: 1601508213606
 nav_order: 2
 ---
+## 0.107.0
+
+### Deprecation Notices
```

Comment:
change wording and add to the release note

`enableHandlerbarTempaltes` has been removed as a configuration option (because it is now always on). No extra action is needed but you will be prompted to remove the deprecated configuration on upgrade